src/form/Label/Label.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { LabelHTMLAttributes, ReactNode } from 'react';
+import { LabelStyle, LabelTextStyle } from './LabelStyle';
+import type { StyleProps } from './LabelStyle';
+import usePassPropsToChildren from '../../utils/hooks/usePassPropsToChildren';
+
+export interface LabelProps extends StyleProps, LabelHTMLAttributes<HTMLLabelElement> {
+	label: ReactNode;
+}
+
+export const Label: React.FunctionComponent<LabelProps> = ({
+	children: childrenProps,
+	label,
+	disabled,
+	position,
```

Comment:
Het label uit ASC (1) ondersteund wel positie van het label: packages/asc-ui/src/components/Label/LabelStyle.ts:66

Reply:
You are right, the story inside asc does not use nesting, therefor the applied style rules do not have the wanted outcome. I guess I missed that one.